spec/libraries/course/assessment/programming_test_case_report_spec.rb

```diff
@@ -202,6 +202,63 @@
     end
   end
 
+  context 'when given a report with meta information attached to test case tags' do
```

Comment:
I think it'll be good to test the `output` attribute as well. For consistency with the rest.